### PR TITLE
FloatCounter, add tostring and implement poolable

### PR DESCRIPTION
**Pull request by fabiitch**
At start i just wanted to add toString method on FloatCounter

And i see the reset method,  so im think 'why not poolable' .

Open to remove the implements Poolable, i just need the toString ^^

Thanks for reading

### CODE REVIEW

**Review by MrStahlfelge — APPROVED** [review]
I don't see a problem here. One could discuss if the toString() method is needed here but it is handy for debugging purposes so why not.

**Review by crykn — APPROVED** [review]

File: gdx/src/com/badlogic/gdx/math/FloatCounter.java [hunk 1]
```diff
@@ -76,4 +79,18 @@ public void reset () {
...
+	public String toString() {
+		return "FloatCounter{" +
+				"count=" + count +
+				", total=" + total +
+				", min=" + min +
+				", max=" + max +
+				", average=" + average +
+				", latest=" + latest +
+				", value=" + value +
+				", mean=" + mean +
```
Comment (obigu) [inline gdx/src/com/badlogic/gdx/math/FloatCounter.java:93]: Can we remove `mean`? `value` already holds the `mean` value.

Comment (fabiitch) [reply gdx/src/com/badlogic/gdx/math/FloatCounter.java]: done

**Comment by MobiDevelop** [comment]
I'm not going to lie, it feels like an odd type to be pooling - so not sure about adding the Poolable interface (even if it happens to have a reset method).

**Review by obigu — APPROVED** [review]
Seems good to me. I don't see an issue with adding the `Poolable` interface even if it's probably going to be rarely pooled.

**Comment by NathanSweet** [comment]
This isn't super important nitpick, but we'd not like this to run away from us and make lots of classes poolable. :)

**Pull request merged**